Source/WebCore/css/ComputedStyleExtractor.h

```diff
@@ -58,6 +58,8 @@ class ComputedStyleExtractor {
     ComputedStyleExtractor(Node*, bool allowVisitedStyle, PseudoId);
     ComputedStyleExtractor(Element*, bool allowVisitedStyle = false);
     ComputedStyleExtractor(Element*, bool allowVisitedStyle, PseudoId);
+    ComputedStyleExtractor(Element&, bool allowVisitedStyle = false);
+    ComputedStyleExtractor(Element&, bool allowVisitedStyle, PseudoId);
```

Comment:
Do we really need this? Is there an important reason to not just add a `&` at the call site?